test/test/test.js

```diff
@@ -23,28 +23,32 @@ describe('Bridge', function () {
   const erc20AppAddress = "0xEDa338E4dC46038493b885327842fD3E301CaB39"
   const tokenAddress = "0x774667629726ec1FaBEbCEc0D9139bD1C8f72a23"
 
-  const ETH_ASSET_ID = "0x00"
 
   before(async function () {
     ethClient = new EthClient("ws://localhost:8545", ethAppAddress, erc20AppAddress);
     subClient = new SubClient("ws://localhost:11144");
     await subClient.connect();
     await ethClient.initialize();
 
+    this.ethAssetId = subClient.api.createType('AssetId', 'ETH');
```

Comment:
@musnit at some point we need to inform the web devs about this new API for asset ids. Can no longer pass "0x000000" as the ETH asset id.

Reply:
yeh, will have to rework a lot of the webapp once we merge in all the channel stuff too - will wait until then and then do all the changes together in a frontend PR.